### PR TITLE
Pptp 2031

### PR DESCRIPTION
**Pull request by ajchilts**
### Description of Work carried through

Brief description of what/why/how.

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [ ] `sbt scalafmt test:scalafmt` was executed 
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [ ] User Acceptance Tests (UAT) were run locally and have passed.
 - [ ] No Accessibility errors/warnings reported by Wave

### CODE REVIEW

File: app/controllers/actions/AuthFunction.scala [hunk 1]
```diff
@@ -64,7 +64,7 @@ class AuthFunction @Inject() (
 
           val maybeInternalId = internalId.getOrElse(
             throw new IllegalArgumentException(
-              s"AuthenticatedIdentifierAction::invokeBlock -  internalId is required"
+              s"AuthenticatedIdentifierAction::invokeBlock -  id is required"
```
Comment (liampan) [inline app/controllers/actions/AuthFunction.scala:67]: lets revert these changes, looks like rind an replace got agro

**Review by lmcewanhmrc — APPROVED** [review]
